### ISSUE TITLE
Messages not displayed properly

### ISSUE THREAD

**Issue by DataBeaver** [mirukana/mirage#35]
After fixing the Python error in #15 I can start Mirage but then it starts spewing QML errors:
```
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Cannot call method 'startsWith' of undefined
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Cannot call method 'startsWith' of undefined
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Type error
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Type error
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Type error
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Type error
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Type error
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Type error
qrc:/src/gui/MainPane/Room.qml:110: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/Daybreak.qml:7: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:90:13: Unable to assign [undefined] to QString
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:89:13: Unable to assign [undefined] to QString
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:91:13: Unable to assign [undefined] to QString
qrc:/src/gui/Utils.qml:167: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/EventDelegate.qml:39: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:14: SyntaxError: JSON.parse: Parse error
qrc:/src/gui/Utils.qml:167: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:266: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:274: SyntaxError: JSON.parse: Parse error
qrc:/src/gui/Pages/Chat/Timeline/Daybreak.qml:7: TypeError: Type error
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:90:13: Unable to assign [undefined] to QString
qrc:/src/gui/Pages/Chat/Timeline/EventContent.qml:89:13: Unable to assign [undefined] to QString
```

All messages in the room are displayed with a question mark avatar and empty text.

I can investigate this further when I have time, but if you have any insights that would be appreciated.

**Comment by mirukana** [comment]
I don't think this is related to #15, the code changes nothing for me. What about #34? How did you install Mirage?

**Comment by DataBeaver** [comment]
I cloned the git repo and compiled from source.  I'm running it directly from the build tree; everything seems to be packed into the executable so I figured its location would not matter.

I was not getting the exception mentioned in that issue, nor any other exceptions.  In fact the only output line before the ones I pasted was:

```
Got library name:  "/usr/lib/x86_64-linux-gnu/qt5/qml/io/thp/pyotherside/libpyothersideplugin.so"
```

However I pulled the fix for that issue and it seems to have fixed my problem as well.  Messages are now displayed correctly.
